packages/glamor-jsxstyle/example/src/jsxstyle.js

```diff
@@ -1,17 +0,0 @@
-import React from 'react'
-import { View, Row, Column } from 'glamor-jsxstyle'
```

Comment:
We need to add these components to glamorous. Right now it's just elements which you can style.